packages/react-vanilla-components/__tests__/components/DropDown.test.tsx

```diff
@@ -19,7 +19,15 @@ const field = {
   fieldType: "drop-down",
   placeholder: "select",
   enum: [1, 2, 3],
-  enumNames: ["option 1", "option 2", "option 3"],
+  enumNames: [{
```

Comment:
please add test cases for richtext

Reply:
There is no richtext for options in dropdown!